Sources/Rendering/Core/VolumeMapper/index.js

```diff
@@ -93,6 +93,16 @@ function vtkVolumeMapper(publicAPI, model) {
     model.anisotropy = vtkMath.clampValue(at, -0.99, 0.99);
     publicAPI.modified();
   };
+
+  publicAPI.setKernelSize = (ks) => {
+    model.kernelSize = vtkMath.floor(vtkMath.clampValue(ks, 1, 32));
```

Comment:
If you have a `const superClass = { ...publicAPI }` at the beginning of the vtkVolumeMapper func block, you can instead do the following:

```
publicAPI.setKernelSize = (ks) => 
  superClass.setKernelSize(vtkMath.floor(vtkMath.clampValue(ks, 1, 32)));`
```

This will use the setGet auto-generated method, which calls modified() for you and returns a boolean as to whether the value changed.

Reply:
Funny how people like different things. I almost added a comment saying to not use superClass like this because what you are doing does not invoke the superclass, but they I saw your comment suggesting to do it this way :-) All good with me, just funny.

Reply:
Yeah, technically there isn't anything involving the superclass, but I never did think of a better name. I just preferred making a copy of the publicAPI up front when augmenting the macro-added functions, and "superclass" was the closest I could get.